.circleci/dockerfiles/centos-8.3

```diff
@@ -0,0 +1,9 @@
+FROM centos:8.3.2011
+
+RUN echo fastestmirror=1 >> /etc/dnf/dnf.conf
```

Comment:
Do we need this for centos 7?

Reply:
Centos7 doesn't have dnf installed by default in the container